### DISCUSSION TITLE
Magick.NET ISOcoatedv2  ColorSpace

### DISCUSSION THREAD

**Discussion by tomasr78** [dlemstra/Magick.NET#1208] [Help] [answered]
Does `Magick.NET` support `ISOcoatedv2` `ColorSpace `by default?

If not, can I load it like this?

```
using (MagickImage image = new MagickImage("input.jpg"))
{
  image.AddProfile(new ColorProfile(@"C:\Path\ISOcoated_v2_eci.icc"));
  image.Write("output.pdf");
}
```

**Comment by dlemstra** [comment, marked as answer]
Not sure what you are asking `ISOcoatedv2` is a color profile and not a color space?

**Comment by tomasr78** [comment]
Yes, I mixed up these two things. Thank you for help.
